common/src/main/kotlin/org/wycliffeassociates/otter/common/domain/audio/AudioConverter.kt

```diff
@@ -39,4 +46,51 @@ class AudioConverter @Inject constructor() {
             jump3r().run(args)
         }
     }
+
+    fun wavToPcm(wavFile: File, pcmFile: File): Completable {
+        return Completable.fromCallable {
+            val wavReader = WavFile(wavFile).reader()
+            val pcmWriter = PcmFile(pcmFile).writer(append = false)
+
+            try {
+                wavReader.open()
+                val buffer = ByteArray(10240)
+                while (wavReader.hasRemaining()) {
+                    val written = wavReader.getPcmBuffer(buffer)
+                    pcmWriter.write(buffer, 0, written)
+                }
+            } finally {
+                cleanup(wavReader, pcmWriter)
+            }
+        }
+    }
+
+    fun pcmToWav(pcmFile: File, wavFile: File): Completable {
+        return Completable.fromCallable {
+            val pcmReader = PcmFile(pcmFile).reader()
+            val wavWriter = WavFile(
+                wavFile,
+                DEFAULT_CHANNELS,
+                DEFAULT_SAMPLE_RATE,
+                DEFAULT_BITS_PER_SAMPLE
+            ).writer(append = false)
+
+            try {
+                pcmReader.open()
+                val buffer = ByteArray(10240)
+                while (pcmReader.hasRemaining()) {
+                    val written = pcmReader.getPcmBuffer(buffer)
+                    wavWriter.write(buffer, 0, written)
+                }
+            } finally {
+                cleanup(pcmReader, wavWriter)
```

Comment:
rather than try/finally, it would probably be better to modify things to allow for .use { }